js/file.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as domTypes from "./dom_types";
```

Comment:
Hmmm... `file` is a bit confusing with `files`.  I wonder if this should be structured so that Web API stuff is in a sub folder where things are a bit less confusing, and `Blob` and `Headers` and `fetch`, etc. can go in there.

Reply:
Indeed, we could have `js/webapi/`, `js/fileapi`, `js/denoapi`, and others. But this is out of the scope of this PR. We can do this later once Event and EventTarget are merged too to prevent merge conflicts/issues with other collaborators working on this.